specifyweb/frontend/js_src/lib/utils/ajax/definitions.ts

```diff
@@ -84,4 +85,6 @@ export const httpCodeToErrorMessage: RR<ValueOf<typeof Http>, string> = {
   [Http.INSUFFICIENT_STORAGE]: `
     This error likely happened because the server has run out of storage space.
   `,
+  [Http.UNPROCESSABLE]:
```

Comment:
could you insert httpCodeToErrorMessage in the same order as in Http?

Reply:
Fixed in f0e372fb81dd0a622c8c7410995e677237b0097f